src/CMakeLists.txt

```diff
@@ -1109,6 +1109,10 @@ if(WITH_LTTNG)
   add_dependencies(vstart tracepoint_libraries)
 endif(WITH_LTTNG)
 
+if(WITH_MGR_DASHBOARD_V2_FRONTEND AND NOT CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|AARCH64|arm|ARM")
```

Comment:
Based on the [current discussion on ceph-devel](https://www.spinics.net/lists/ceph-devel/msg40367.html), would it make sense to exclude non-Linux platforms for the time being, too?

Reply:
I'd rather do `AND CMAKE_SYSTEM_PROCESSOR MATCHES "x86_64"` because if we just exclude ARM we can expect problems on e.g. s390x, power8, i586 etc.